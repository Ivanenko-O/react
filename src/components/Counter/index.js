import React, { Component } from 'react';
import PropTypes from 'prop-types';

export default class Counter extends Component {
    static propTypes = {
        lengthPost: PropTypes.number.isRequired
    }

    render() {
        const { lengthPost } = this.props;

        return (
            <span> { lengthPost } </span>
        )
    }
}
