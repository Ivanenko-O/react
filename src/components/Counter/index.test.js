import React from 'react';
import dom from 'react-test-renderer';

import Counter from './';

const renderTree = dom.create(<Counter lengthPost = { 3 } />).toJSON();

test('Counter component should correspond to its snapshot countrepart', () => {
    expect(renderTree).toMatchSnapshot();
});
