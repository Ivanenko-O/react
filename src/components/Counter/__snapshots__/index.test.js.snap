// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should correspond to its snapshot countrepart 1`] = `
<span>
   
  3
   
</span>
`;
